### PR TITLE
Sentry errors

### PR DESCRIPTION
**Pull request by annehaley**

### CODE REVIEW

File: web_client/src/components/ControlPanel.vue [hunk 1]
```diff
@@ -411,7 +411,10 @@ export default {
                           :decision="decision"
                         />
                         <div
-                          v-if="currentViewData.scanDecisions.length === 0"
+                          v-if="
+                            currentViewData?.scanDecisions === undefined
+                              || currentViewData?.scanDecisions?.length === 0
```
Comment (zachmullen) [inline web_client/src/components/ControlPanel.vue]:
```suggestion
                            !currentViewData?.scanDecisions?.length
```

Is this equivalent?

Comment (annehaley) [reply web_client/src/components/ControlPanel.vue]: Yes, thanks for the short version!

Comment (annehaley) [reply web_client/src/components/ControlPanel.vue]: Actually, I'm discovering that neither works; we cannot use the `?` operator like this within the template. https://github.com/vuejs/vue/issues/11088.

Comment (zachmullen) [reply web_client/src/components/ControlPanel.vue]: If we did want to use that syntax, we could throw it into a computed property and use that in the template.

**Review by zachmullen — APPROVED** [review]

**Comment by zachmullen** [comment]
LGTM. I think moving a lot of this to typescript would catch a lot of these errors at build time.

**Pull request merged**